### DISCUSSION TITLE
iPad 2,4 Downgrade 6.1.3 + JB

### DISCUSSION THREAD

**Discussion by AlexanderQueen** [LukeZGD/Legacy-iOS-Kit#594] [General]
As i saw, we don't have an option in Legacy Kit for iPad 2,4 to downgrade to 6.1.3 only 8.4.1. 
1. In current case, what does a blob mean? its'a full downgrade version or smth cut? 
2. I have downloaded firmware 6.1.3 for iPad 2,4 and i was thinking if its possible to install it in DFU mode thru legacy kit or chance of brick is higher?

**Comment by LukeZGD** [comment]
1. Short explanation: https://github.com/LukeZGD/Legacy-iOS-Kit/wiki/Saving-SHSH-blobs
Longer explanation with analogy: https://www.reddit.com/r/jailbreak/comments/p0giui/tip_a_trip_to_obristan_blobs_explained_with/

2. Do you have 6.1.3 blobs? If not, then you cannot downgrade untethered. You will need to settle with dualbooting using Coolbooter, or tethered downgrade. For tethered downgrade, you need an Arduino or Pi Pico for checkm8-a5. https://github.com/LukeZGD/Legacy-iOS-Kit/wiki/Restore-Downgrade

**Comment by AlexanderQueen** [comment]
First of all i want to try to use this method:  Other (powdersn0w 7.1.x blobs). Will it work? As i saw it must work for all devices from iOS 5.1 till the last 9.3.5. I'm on 9.3.5. I will try to create a blob from 9.3.5 and then downgrade to 6.1.3
![image](https://github.com/user-attachments/assets/75eb6bbb-3aaf-4f39-a2f1-f71b475f097a)

Or should i use : "Other (Use SHSH Blobs)|

P.S Bad idea. Not working. And where to find a signed 6.1.3 firmware is a  problem...

**Comment by AlexanderQueen** [comment]
Therefore, i did coolboot for ios 8.4.1 and 6.1.3. The main problem is that from iOS 8.4.1 USB charging for iPad not working. Tested it on Win and Linux. The charging icon holds a second and then writing not charging. Any idea how to fix it?
